Problem1.py

```diff
@@ -0,0 +1,35 @@
+# Time Complexity : O(n)
+# Space Complexity : Confused between O(n) and O(n^2). HELP NEEDED.......
```

Comment:
what are you taking as n here?

Reply:
n is the number of elements in the matrix, i.e rows*colums.

Reply:
yup